### ISSUE TITLE
Allow immediate application of edits

### ISSUE THREAD

**Issue by jhillyerd** [twpayne/chezmoi#28]
It would be nice to optionally have edits applied immediately.

Not sure what the best implementation would be, but some ideas:

- `chezmoi edit --apply`
- a franken-command like `chezmoi eda` (edit->diff->apply) that potentially prompts "Apply Y/n?" after printing the diff?

**Comment by twpayne** [comment]
I agree. A few thoughts:

* I prefer flags to franken-commands, for fraken-commands we could add an alias system (like [git's alias](https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases). I'll open an issue for this.

 * The "diff" and "apply" should only touch the affected file.

 * For the prompt, I'd add a `--prompt` option to explicitly make it interactive.

With the above we get the verbose `chezmoi edit --diff --apply --prompt` but if we add an alias system then we can simulate the proposed `chezmoi eda` with the alias `eda = edit --diff --apply --prompt`.

**Comment by jhillyerd** [comment]
> The "diff" and "apply" should only touch the affected file.

Good point.  I'm on board with flags, even before you chezmoi gets built-in alias support, folks can still use their shells alias system.
